web/themes/new_weather_theme/assets/sass/components/_radar.scss

```diff
@@ -17,3 +17,45 @@
   border: 3px solid white;
   border-radius: 100%;
 }
+
+.wx-radar-legend {
+  h4.visual-h3 {
+    @include u-text("normal");
+
+    margin-top: 0;
+  }
+
+  .wx-radar-legend__row {
+    th {
+      vertical-align: top;
+    }
+
+    td {
+      padding: 0;
+      vertical-align: top;
+
+      // The cell needs a height in order for
+      // its contents to have something to be
+      // relative to. It's an older hack, sir,
+      // but it checks out.
+      min-height: 1px;
+    }
+
+    .wx-radar-legend__intensity {
+      // Use padding so that it goes inside
+      // the border, effectively pushing
+      // text content left and down from the
+      // gradient.
+      padding: 0.5rem 1rem;
```

Comment:
We've been using USWDS units for most sizing/spacing so think it'd be worth replacing these two values with `units(1)` and `units(2)` respectively.

Reply:
Oops! Thanks, good catch. I'd meant to go back and replace as much as I could with USWDS functions, but I got too eager... 😛